tst/hashmap.tst

```diff
@@ -235,8 +235,8 @@ Error, <key> must not be equal to 'fail'
 # a positional object which isn't a hash map
 gap> IsPositionalObjectRep(infinity);
 true
-gap> DS_Hash_Value(infinity, 1);
-Error, <ht> must be a hashmap object (not a object (positional))
+gap> DS_Hash_Value(x -> x, 1);
+Error, <ht> must be a hashmap object (not a function)
```

Comment:
Actually, that test intentionally used a positional object that's not a hashmap (see also the preceding lines). Of course we can drop it, but then this test and the one before it should be deleted

Reply:
You are right, I should have read the context.